backend/manager/modules/restapi/jaxrs/src/main/java/org/ovirt/engine/api/restapi/resource/BackendStorageDomainVmResource.java

```diff
@@ -104,6 +104,7 @@ public Response register(Action action) {
             params.setClusterId(getClusterId(action));
         }
         params.setImagesExistOnTargetStorageDomain(true);
+        action.setAsync(false);
```

Comment:
Can you please describe what are the considerations behind setting it hard-coded like this vs leaving it to the user? The user may set <async>false</async> in any action. It would seem to me that the only justification for deciding this for the user, is if this action never, in no context, makes sense with async=true. Is this indeed the case?

Reply:
yeah, that might be too aggressive, there's not much sense in executing it as async since the lease is supposed to exist but it is important to change the default to sync, not necessarily to force it to be a sync operation no matter what the user specified. I'll try to change it

Reply:
changed only the default setting to false, which enables the caller to turn it on if they wish to. thanks @oliel , good point